mux/bindings/conformance/runner.py

```
@@ -0,0 +1,325 @@
+    passed = 0
+    failed = 0
+    skipped = 0
+    try:
+        data = json.loads(FIXTURES.read_text())
+        defaults = data.get("defaults", {})
+        for fixture in data.get("fixtures", []):
+            try:
+                run_fixture(fixture, defaults, server["socket"])
+            except FixtureSkipped as exc:
+                skipped += 1
+                print(f"SKIP {fixture.get('name', '<unnamed>')}: {exc}")
```

Ruff (0.15.20)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)